aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -182,6 +182,12 @@ module aptos_framework::transaction_context {
         payload.entry_function_payload
     }
 
+    /// Returns the hash of the current raw transaction.
```

Comment:
Can we add documentation to explain the difference between this one and the other ones. May be say the other ones shouldn't be used?